### DISCUSSION TITLE
Could I use colossalAI on Windows?

### DISCUSSION THREAD

**Discussion by cccrice** [hpcaitech/ColossalAI#991] [Community | Q&A] [answered]
I got "ImportError: cannot import name 'RRef'" when I tried to run train_with_cifar10.py on Windows, my pytorch version is 1.9.0 + cuda 11.1

**Comment by ver217** [comment, marked as answer]
Hi, we currently do not support Windows. However, you can use [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) which supports ColossalAI well.
